examples/httpx_client.py

```diff
@@ -23,22 +25,19 @@
 logger = logging.getLogger("client")
 
 
-class H3Transport(QuicConnectionProtocol, httpcore.AsyncHTTPTransport):
+class H3Transport(QuicConnectionProtocol, httpx.AsyncHTTPTransport):
```

Comment:
The base class for custom Async transports is `AsyncBaseTransport`. You'll want to subclass that, rather than `AsyncHTTPTransport`.

You don't need to call `__init__` for it, as it's (effectively) just an interface.

Reply:
Ah! That's helpful. Then I bet the other example is working fine.